specs/commandline/verify.md

```diff
@@ -25,9 +25,6 @@ The artifact was signed with the following user metadata.
 KEY    VALUE
 <key>  <value>
 ```
-> [!NOTE]
-> This command is for verifying OCI artifacts only. Use `notation blob verify` command for verifying arbitrary blobs.
```

Comment:
This part was not removed in `v1.1.1`. Please send out a PR to `release-1.1` to fix it.